src/modules/mysql.nix

```diff
@@ -120,19 +120,20 @@ in
       cfg.package
     ];
 
-    env.MYSQLDATA = config.env.DEVENV_STATE + "/mysql";
-    env.MYSQLSOCKET = config.env.DEVENV_STATE + "/mysql.sock";
+    env.MYSQL_HOME = config.env.DEVENV_STATE + "/mysql";
+    env.MYSQL_UNIX_PORT = config.env.DEVENV_STATE + "/mysql.sock";
+    env.MYSQL_TCP_PORT = mkIf (cfg.settings.mysqld ? port) (toString cfg.settings.mysqld.port);
 
     scripts.mysql.exec = ''
-      exec ${cfg.package}/bin/mysql ${mysqlOptions} --socket=$MYSQLSOCKET $@
+      exec ${cfg.package}/bin/mysql ${mysqlOptions} "$@"
```

Comment:
If we don't need to set `--socket` we can remove all scripts related to mysql

Reply:
`my.cnf` is not in `MYSQL_HOME` yet. If we copy or link it to `MYSQL_HOME` then the wrappers can be removed.

Maybe copy/linking it right before starting mysqld could be an option?

Reply:
Yea we could link it while starting the MySQL server. 👍

Reply:
I'll look into doing that in a separate PR.